### ISSUE TITLE
[question] minLandVisible seems not to be effective

### ISSUE THREAD

**Issue by chris-aeviator** [flaviostutz/sentinelloader#9]
Could you please explain the minLandVisible parameter?

Is it 0 to 100% expressed as 0 to 1 or is it percentage 0 to 100 ?

EDIT: 

it seems to not do anything except giving a warning

I expect an image to be not included into the results when passing a certain threshold (e.g. I want to have 90% land visible) to not have false NDVI values in my results. No matter what I set the `minLandVisible` parameter, the result will contain cloud images

![image](https://user-images.githubusercontent.com/11522213/107356145-bc4f7d80-6ad0-11eb-8ece-15860d87078e.png)

**Comment by flaviostutz** [comment]
You are right. When the sample has less visible land than minLandVisible (because of clouds) it is not meant to be returned in the results.
"minLandVisible" is meant to be from 0-1, like 0.3, 0.5...

Could you share the notebook code you used for generating those samples?
